iwyu_ast_util.cc

```diff
@@ -660,6 +674,19 @@ bool IsTemplatizedFunctionDecl(const FunctionDecl* decl) {
 }
 
 bool HasImplicitConversionCtor(const CXXRecordDecl* cxx_class) {
+  // Member functions of class templates aren't instantiated unless used,
```

Comment:
Optional: This looks like a bug that just happened to surface now. Is it possible to commit it separately with an isolated testcase before this PR?

Reply:
#1314 merged, thanks!